src/Compatibility/Microsoft.DotNet.ApiCompatibility/Logging/Suppression.cs

```diff
@@ -44,11 +49,9 @@ public Suppression(string diagnosticId)
         }
 
         /// <summary>
-        /// <see langword="true"/> if the suppression is to be applied to a baseline validation. <see langword="false"/> otherwise.
+        /// It only makes sense to serialize IsBaselineSuppression when is true, if it is off, no need to have it on the file.
```

Comment:
NIT: I would just add a quick blurb here explaining why is this method used and where, potentially pointing to xml docs. 

In short, this method is defined here because when the xml serializer is serializing this type, it uses reflection to call into this method in order to figure out whether or not the value of IsBaselineSuppression should be serialized. The naming here is also important, because the serializer will look for a method called: `ShouldSerialize<nameOfTheProperty>` and if it exists, it will call it and figure out if it has to serialize it or not. Anyway, just adding a small blurb explaining would be good so that it doesn't get removed or renamed by mistake in the future thinking that we aren't actually using it.

Reply:
I believe this is where the code is performing that check: https://github.com/dotnet/runtime/blob/c80a49d151b5e642be419067adf401a365b624b7/src/libraries/System.Private.Xml/src/System/Xml/Serialization/ReflectionXmlSerializationWriter.cs#L594-L599